Sources/SwiftKuery/Table.swift

```diff
@@ -159,16 +288,50 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, a column for id is initialized and a `Table` instance called "personTable" is created.
+     The personTable primary key is then set to be "idColumn".
+     ```swift
+     let idColumn = Column("id", Int32.self, notNull: true)
+     public class PersonTable : Table {
+        let tableName = "personTable"
+        let id = idColumn
+        let firstName = Column("firstName", String.self, notNull: true)
+        let lastName = Column("lastName", String.self, notNull: true)
+     }
+     var personTable = PersonTable()
+     personTable = personTable.primaryKey(idColumn)
+     ```
+     */
+    /// - Parameter columns: Single Column that constitute the primary key.
```

Comment:
A single column that constitutes the primary key.
(You can capitalize and backtick the Column word if you prefer)